lib/logstash/filters/grok.rb

```diff
@@ -331,6 +326,24 @@ def close
 
     private
 
+    # The default pattern paths, depending on environment.
+    def patterns_path
+      patterns_path = []
+      case (ecs_compatibility rescue :disabled) # TODO
```

Comment:
this rescue clause gives me pause. Or is it just a stub for the compatibility support's inclusion being commented out?

Reply:
just a stub - there's TODOs commented parts all over the place - till the ecs support gem is pushed.

Reply:
It's aliiiiive -> https://rubygems.org/gems/logstash-mixin-ecs_compatibility_support